inst/examples/NicolasPred.R

```diff
@@ -1,50 +1,34 @@
-#install.packages("devtools")
-#devtools::install_github("choisy/gamar")
-library(gamar)
-defpath("/Applications/Gama.app")
-myModelPath <- "/Users/jdz/git/gamar/inst/examples"
-#(myModelFile <- paste0(system.file("examples",package="gamar"),"/cluster_ind.gaml"))
-(myModelFile <- paste0(myModelPath,"/NicolasPred.gaml"))
+# install and load the package
+devtools::install_github("r-and-gama/rama")
+library(rama)
 
-(experimentPREY <- getmodelparameter(modelfile = myModelFile,experimentname = "prey_predator"))
-(getparameternames(experimentPREY))
+# if necessary to configure GAMA path
+# defpath("/Applications/Gama.app")
 
-action_distance <- data.frame("action_distance"=c(100,200,300,400,500,600,700,800,900,1000,1500,2000,2500,3000,3500,4000,5500,6000,7000,8000,9000,10000,12000,14000,16000,20000,25000,30000,40000,50000))
-environment_size <-  data.frame("environment_size"=c(500,10000,15000,20000,30000,40000,50000))
-all_seed <-  data.frame("mseed"=seq(1,30,1))
+# to load the experiment
+exp <- load_experiment("prey_predator",
+                       system.file("examples", "NicolasPred.gaml",
+                                   package = "rama"))
 
 
-simulation_duration <- 5000;
+
+
+# to set the experiment
+exp$p_danger_distance <-  c(100, 200, 300, 400, 500, 600, 700, 800, 900,
+                            1000, 1500, 2000, 2500, 3000, 3500, 4000, 5500,
+                            6000, 7000, 8000, 9000, 10000, 12000, 14000, 16000,
+                            20000, 25000, 30000, 40000, 50000)
+exp$p_environment_size <- c(500, 10000, 15000, 20000, 30000, 40000, 50000)
```

Comment:
Can we write `c(1000 * seq(5, 20, 5), 10000 * 3:5)` instead?

Reply:
Done.